### ISSUE TITLE
Cannot not load the asset file

### ISSUE THREAD

**Issue by kenfly51** [JimmyLaurent/hcaptcha-solver#19]
![Screenshot 2021-07-19 235306](https://user-images.githubusercontent.com/3773831/126197510-55abb167-190f-4e3b-92a1-7714b1967759.jpg)

Just follow the guide, but I got this error. It seems like the assets file is out of date?

**Comment by jairoxyz** [comment]
Asset file is now at https://newassets.hcaptcha.com/c/420b6d0a/hsl.js

Sorry wrong url. Try this one.

**Comment by fstackgeek** [comment]
@jairoxyz I did this but it doesn't work. Did you manage to get it to work? can you explain what did you do exactly?

**Comment by jairoxyz** [comment]
@amiiir do you have a site to test with? Please try this url, I accidentally copied the wrong one. https://newassets.hcaptcha.com/c/420b6d0a/hsl.js

**Comment by fstackgeek** [comment]
I used this new asset, but I got the same error:
```{
  c: {
    type: 'hsj',
    req: '...'
  },
  success: false,
  'error-codes': []
}
```
This is what is returned in response, but the code expects something else with the properties `{key, tasklist, request_type}`

**Comment by jairoxyz** [comment]
Yes, the thing is that uuid() doesn't work for sitekey, so I used the one scraped from the site and I get a good response from getcaptcha but then the last call to checkcaptcha returns 403. Not sure why yet.

**Comment by fstackgeek** [comment]
@jairoxyz  I don't have much experience with this package and the whole mechanism of how hCaptcha works. If you could get it to work, please post your solution here. Thank you so much

**Comment by jairoxyz** [comment]
I think best is to look into this implementation: https://github.com/aw1875/puppeteer-hcaptcha

**Comment by zaniniarthur** [comment]
> Usei este novo ativo, mas recebi o mesmo erro:
> 
> ```
>   c: {
>     type: 'hsj',
>     req: '...'
>   },
>   success: false,
>   'error-codes': []
> }
> ```
> 
> Isso é o que é retornado em resposta, mas o código espera algo diferente com as propriedades `{key, tasklist, request_type}`

Estou com essa mesma falha... alguem conseguiu resolver?

**Comment by pythonDLL** [comment]
you guys atleast try to read the code bruh its just random true or false thing it will never work
